### PR TITLE
Update lighting distribution.

### PR DESCRIPTION
**Pull request by afontani**
## Pull Request Description

The old lighting distribution is weighted too heavily towards incandescent bulbs. New distribution data is from 2015 in the 2015 U.S. Lighting Market Characterization study. Option= 100% Incandescent include Incandescent, HID, Halogen, and Other. Option=100% CFL include CFL and LFL. Option=100% LED include LED.

## Checklist

Not all may apply:

- [x] Unit tests have been added or updated
- [x] All rake tasks have been run, and pass
- [ ] PAT project measures/outputs have been updated
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [x] No unexpected regression test changes
- [ ] All tests are passing (green) on circleci
- [x] The [changelog](https://github.com/NREL/OpenStudio-BuildStock/blob/master/CHANGELOG.md) has been updated appropriately
- [x] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

**Comment by afontani** [comment]
@joseph-robertson This is ready for PAT updates.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 Features
 - Update the multifamily project with a Geometry Wall Type tsv file for sampling between wood stud and masonry walls ([#382](https://github.com/NREL/OpenStudio-BuildStock/pull/382))
+- Update the multifamily project with a more up to date lighting stock distribution. ([#392](https://github.com/NREL/OpenStudio-BuildStock/pull/392))
```
Comment (joseph-robertson) [inline CHANGELOG.md]: Probably want to update the changelog to mention changes to the single-family detached project as well.

File: Rakefile [hunk 2]
```diff
@@ -481,10 +481,13 @@ def check_parameter_file_format(tsvpath, n_deps, name)
   # For each line in file
   i = 1
   File.read(tsvpath, mode: "rb").each_line do |line|
+    next if line[0].start_with? "C"
```
Comment (joseph-robertson) [inline Rakefile]: Won't we miss lines, e.g., when `Dependency=HVAC System Cooling Type` has a row starting with "Central"?

File: data/other/tsv_maker.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            df (pandas.DataFrame): the dataframe that the non "Dependency=" columns are converted to floats
+        Returns:
+            df (pandas.DataFrame): the same dataframe as the input argument except with non dependency columns as floats
+        """
+        matching = [s for s in df.columns.values if not "Dependency=" in s]
+        for col in matching:
+            df[col] = df[col].astype(float)
+        return df
+
+    def export_and_tag(self, df, filepath, project):
```
Comment (joseph-robertson) [inline data/other/tsv_maker.py]: I feel like we should try to inherit the existing `TSVMaker` base class as much as possible. See [this](https://github.com/NREL/OpenStudio-BuildStock/blob/master/data/recs/2009/tsv_maker.py#L21) as an example.

Comment (afontani) [reply data/other/tsv_maker.py]: @joseph-robertson Thanks for the comments. I refactored the RECS 2009, RECS 2015, and the OtherSources tsv_makers. I added the capability to pull the data from s3. Let me know if this cleans things up.

**Pull request merged**